.github/tool_shlock_helper.sh

```diff
@@ -73,7 +73,6 @@ while [ $# -gt $MINPARAMS ]; do    # Until you run out of parameters . . .
 
 case "$1" in
   -p|--pid) shift ; export PID_VALUE="${1}" ;;
-  -v|--verbose) IS_VERBOSE_MODE=1 ;;
```

Comment:
🤔 too bad this is not used. Verbose mode is a staple of the more common options one expects.
Unused still means waste of code though.